### ISSUE TITLE
segfault in test suite (Mix_Music)

### ISSUE THREAD

**Issue by remicollet** [kea/php-sdl-mixer#6]
php -n -d extension=sdl -d extension=modules/sdl_mixer.so tests/003_loadMUS.php

```
$ php -n -d error_reporting=24575 -d extension=sdl -d extension=modules/sdl_mixer.so tests/003_loadMUS.php
bool(false)
string(77) "Couldn't open '/work/GIT/pecl-and-ext/sdl_mixer/tests/boing_doesnt_exist.ogg'"
bool(true)
Segmentation fault (core dumped)

$ php -n -d error_reporting=24575 -d extension=sdl -d extension=modules/sdl_mixer.so tests/004_audio_query.php
int(1)
int(44100)
int(32784)
int(2)
string(0) ""
Segmentation fault (core dumped)
```

tests 001 and 002 are OK.

P.S. can't reproduced in gdb... hard to find what happen, but seems related to shutdown process and object destruction

**Comment by remicollet** [comment]
Should be fixed by second commit in pr #5

**Comment by kea** [comment]
I confirm that your pr fixes it. Thanks
